auth-web/src/components/auth/staff/review-task/AccountStatus.vue

```diff
@@ -96,6 +104,10 @@ export default class AccountStatusTab extends Vue {
     return this.taskDetails.status === TaskStatus.HOLD
   }
 
+  private get isTaskRejected (): boolean {
+    return this.taskDetails.relationshipStatus === this.TaskRelationshipStatusEnum.REJECTED
```

Comment:
If you're changing these a bit, feel free to convert to composition-api. I know you're busy working on assets stuff though haha.

Reply:
I'll have a look, i don't mind!